README.md

```diff
@@ -101,6 +104,20 @@ If this argument is not provided, the default value of `2` will be used. This is
 
 An example input json file that contains the required inputs and these optional inputs can be found [here](https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/dockstore-workflow-rnaseq/7d64748a3a6d7cc8cfd9f30fc43c1b9bc79b3b3f/jobs/test-paired-fastq/job.json)
 
+In addition, you may optionally specify the following parameters for the STAR alignment:
```

Comment:
do you want to make a comment here that this is typically used for miRNA?

Reply:
Good idea, I'll note that it is highly recommended to supply these arguments for miRNA